server/src/models/Website.js

```diff
@@ -17,6 +17,14 @@ class Website extends Model {
             },
         }
     }
+
+    static async findAll() {
+        return this.query();
+      }
+    
+      static async saveAll(data) {
+        return this.query().insert(data);
+      }
```

Comment:
- fix the indentation
- additionally, you define these methods but never use them. I am okay with you making this class methods, but only if they are used. Please delete them until you are actually using them somewhere